app/components/PillLabelBox.tsx

```diff
@@ -0,0 +1,62 @@
+import { tw } from '@helpwave/common/twind/index'
+
+export type PillLabelBoxProps = {
+  unscheduledTasks: number,
```

Comment:
We should remove the suffix task. In the future, we should have a separate type called "task". Can be confusing.